tools/RAiDER/delayFcns.py

```diff
@@ -167,7 +155,64 @@ def make_interpolator(xs, ys, zs, data):
     )
 
 
-def process_chunk(k, chunkInds, SP, SLV, in_shape, stepSize, ifWet, ifHydro, max_len, wm_file):
+def chunk(chunkSize, in_shape):
+    '''
+    Create a set of indices to use as chunks
+    '''
+    startInds =  makeChunkStartInds(chunkSize, in_shape)
+    chunkInds = makeChunksFromInds(startInds, chunkSize, in_shape)
+    return chunkInds
+
+
+def makeChunksFromInds(startInd, chunkSize, in_shape):
```

Comment:
Since `makeChunkStartInds` is called first above, I would expect it to be defined first as well. This helps with reading the code, because the way it is now, I have to skip down and back up instead of just being able to read straight down.